README.md

```diff
@@ -93,45 +93,115 @@ Brainflow:
+| osc_heart_bpm | BFI/Biometrics/HeartBeatsPerMinute |
+| osc_heart_bps | BFI/Biometrics/HeartBeatsPerSecond |
+| osc_oxygen_percent | BFI/Biometrics/OxygenPercent |
+| osc_respiration_bpm | BFI/Biometrics/BreathsPerMinute |
+| osc_band_power_left_alpha | BFI/PwrBand/Left/Alpha |
+| osc_band_power_left_beta | BFI/PwrBand/Left/Beta |
+| osc_band_power_left_theta | BFI/PwrBand/Left/Theta |
+| osc_band_power_left_delta | BFI/PwrBand/Left/Delta |
+| osc_band_power_left_gamma | BFI/PwrBand/Left/Gamma |
+| osc_band_power_right_alpha | BFI/PwrBand/Right/Alpha |
+| osc_band_power_right_beta | BFI/PwrBand/Right/Beta |
+| osc_band_power_right_theta | BFI/PwrBand/Right/Theta |
+| osc_band_power_right_delta | BFI/PwrBand/Right/Delta |
+| osc_band_power_right_gamma | BFI/PwrBand/Right/Gamma |
+| osc_band_power_avg_alpha | BFI/PwrBand/Avg/Alpha |
+| osc_band_power_avg_beta | BFI/PwrBand/Avg/Beta |
+| osc_band_power_avg_theta | BFI/PwrBand/Avg/Theta |
+| osc_band_power_avg_delta | BFI/PwrBand/Avg/Delta |
+| osc_band_power_avg_gamma | BFI/PwrBand/Avg/Gamma |
+| osc_battery_lvl | BFI/Info/BatteryLevel |
+| osc_is_connected | BFI/Info/DeviceConnected |
+| osc_time_diff | BFI/Info/SecondsSinceLastUpdate |
+| HueShift | BFI/Addons/HueShift |
+
+If you still want to use the old parameter scheme, you can switch to them by adding the `--use-old-reporter` launch argument. Be aware that this will be deprecated in the future and an official announcement will be made for its sunset.
+
+## Parameter Descriptions
+### Utility
+These utility Parameters give basic information about your device and BFiVRC
+| Parameter | Description | Type |
+| ------ | ----- | ----- |
+| BFI/Info/VersionMajor | The major version number of BFiVRC | Int |
+| BFI/Info/VersionMinor | The minor version number of BFiVRC | Int |
+| BFI/Info/SecondsSinceLastUpdate | The refresh rate of BFiVRCs data stream | Float |
+| BFI/Info/DeviceConnected | The connection status of your device to BFiVRC | Bool |
+| BFI/Info/BatterySupported | If your device supports sending battery status to BFiVRC | Bool |
+| BFI/Info/BatteryLevel | The current charge status of your devices battery | Float |
+
+### Neurofeedback
+These Parameters are calculated based on your current mental state and make use of the full signed positive and negative float range.
+| Parameter | Description | Type | Range |
+| ------ | ----- | ----- | ----- |
+| BFI/NeuroFB/FocusLeft | Left Unfocused to Focused | Float | [-1.0, 1.0] |
+| BFI/NeuroFB/FocusRight | Right Unfocused to Focused | Float | [-1.0, 1.0] |
+| BFI/NeuroFB/FocusAvg | Unfocused to Focused | Float | [-1.0, 1.0] |
+| BFI/NeuroFB/RelaxLeft | Left Excited to Relaxed | Float | [-1.0, 1.0] |
+| BFI/NeuroFB/RelaxRight | Right Excited to Relaxed | Float | [-1.0, 1.0] |
+| BFI/NeuroFB/RelaxAvg | Excited to Relaxed | Float | [-1.0, 1.0] |
+
+These are the same Neurofeedback scores remapped to positive floats for easier use with prefabs, animations, and avatar smoothing

```

Comment:
Change to:

These are the same Neurofeedback scores remapped to the positive 0 to 1 range for cases where it may be required.